packages/Filter/src/components/Item/Item.js

```diff
@@ -80,31 +81,29 @@ function Item(props) {
     }
   }
 
-  function handleChangeBooleanFilterValue(event) {
-    onChangeFilter(types.changeTypes.FILTER_VALUE, { id, value: event.target.value === "true" });
+  function handleChangeBooleanFilterValue(index, options) {
+    onChangeFilter(types.changeTypes.FILTER_VALUE, { id, value: options[index].value === "true" });
   }
 
-  function handleChangeSingleSelectFilterValue(event) {
-    onChangeFilter(types.changeTypes.FILTER_VALUE, { id, value: event.target.value });
+  function handleChangeSingleSelectFilterValue(index, options) {
+    onChangeFilter(types.changeTypes.FILTER_VALUE, { id, value: options[index].value });
   }
 
   function renderRuleField() {
     switch (selectedColumnType) {
```

Comment:
curious why to use a switch for two options?

Reply:
Maybe when I worked on it I thought there might be many column types? To be honest I don't recall, or maybe just a simple mistake 😅